doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_topology.rst

```diff
@@ -0,0 +1,475 @@
+
+Radio Domain
+============
+
+In the radio domain, the following three SUIT manifests are supported:
+
+* Two local radio manifests
+* A recovery manifest
+
+Secure Domain and System Controller
+===================================
+
+The Secure Domain and the System Controller support the following manifests:
+
+* The Nordic top manifest
+* A System Controller manifest
+* A Secure Domain manifest
+
+The Secure Domain Manifest is responsible solely for the delivery and initiation of the update process for the Secure Domain Firmware and Secure Domain Recovery Firmware images.
+In the following diagrams, the Secure Domain Manifest is labeled as ``SDFW`` or ``SDFW_RECOVERY``.
+
+.. figure:: ../../../../../../../modules/doc-internal/sw_arch/suit/images/nrf54h20_suit_manifests.png
+
+   SUIT manifests supported by the nRF54H20 SoC in the most complex predicted case
+
+This diagram illustrates the most complex predicted configuration.
+You can choose a subset of the manifests supported by the nRF54H20.
+
+.. figure:: ../../../../../../../modules/doc-internal/sw_arch/suit/images/nrf54h20_suit_manifests_minimal.png
+
+   SUIT manifests enabled in nRF54H20, Application and Radio images, without recovery support
+
+This diagram illustrates the minimal predicted configuration that supports the application and radio domains.
+
+Properties associated to SUIT manifest
+**************************************
+
+.. figure:: ../../../../../../../modules/doc-internal/sw_arch/suit/images/suit_manifest_properties.png
+
+   An example of property values of the SUIT Application Local Manifest
+
+This diagram presents a set of properties associated with the SUIT manifest type, along with *example* values.
+These properties are not part of the manifest itself but are configuration parameters stored on the device, defining the boundaries for allowable manifest behavior.
+The properties covered in this document include:
+
+* The property that enables the manifest
+* The vendor ID
+* The manifest class ID
+* The downgrade prevention policy
+* The independent updateability policy
+* The key ID ranges for SUIT manifest verification and signature verification policy
+* The property granting the ability to operate on device resources
+
+The properties of Nordic-controlled manifests are hard-coded into the SDFW and cannot be modified.
+However, some properties of user-customizable manifests are configurable using the `Manifest provisioning information data structure`_.
+
+Vendor ID and Manifest Class ID
+============================
+
+Vendor ID and Manifest Class ID allow users to specify the Vendor (for example, ``Nordic``, or ``ACME Corp``) and the Manifest Class.
+In other words, they define the intended usage of the manifest, such as the manifest controlling firmware images associated with the Application Domain.
```

Comment:
Should "Application Domain" be "application domain"?

Reply:
non capitalized